tests/prague/eip2537_bls_12_381_precompiles/test_bls12_g2msm.py

```diff
@@ -20,7 +20,24 @@
 ]
 
 
-@pytest.mark.parametrize("input_data,expected_output", vectors_from_file("multiexp_G2_bls.json"))
```

Comment:
are this all vectors from that file? 
yes it is much nicer to have this at code.